crates/noirc_evaluator/src/ssa/function.rs

```diff
@@ -17,51 +16,57 @@ use super::{
     ssa_form,
 };
 
+#[derive(Clone, Debug, PartialEq, Copy)]
+pub struct FuncIndex(usize);
+
+impl FuncIndex {
+    pub fn new(idx: usize) -> FuncIndex {
+        FuncIndex(idx)
+    }
+
+    pub fn get_index(&self) -> usize {
+        self.0
```

Comment:
Instead of providing wrapping and unwrapping functions we can make the inner usize field pub instead

Reply:
Done